node/modules/client.go

```diff
@@ -108,15 +107,9 @@ func NewClientDatastore(ds dtypes.MetadataDS) dtypes.ClientDatastore {
 	return namespace.Wrap(ds, datastore.NewKey("/deals/client"))
 }
 
-type ClientDealFunds funds.DealFunds
-
-func NewClientDealFunds(ds dtypes.MetadataDS) (ClientDealFunds, error) {
-	return funds.NewDealFunds(ds, datastore.NewKey("/marketfunds/client"))
```

Comment:
Warning: This the datastore you probably need to exact a value from for the fund manager -- also, I was thinking about this last night -- it should be an Add, rather than a Set, because in the case of a miner, you want to add both values (a bug in the previous version -- a storage miner who also made deals would not have gotten the correct behavcior our of the previous DealFunds tracker in markets)